app/views/items/show.html.haml

```diff
@@ -83,6 +87,3 @@
     /   %h5
     /     %a{:href => ""} Smithsonian Civil War Collection
     /   %p Description of this collection lorem ipsum dolor sit amet, consectetur adipiscing elit. In ac est leo. Cras sollicitudin volutpat nibh in blandit. Morbi ornare quam quis tortor pretium at hendrerit magna molestie. Cras facilisis, nulla non lobortis varius, tellus nibh elementum nibh, non accumsan augue augue ut augue.
-
-:javascript
-  #{analytics_track_event "View Item : #{@item.try(:provider)}", "#{@item.try(:contributing_institution)}", "#{@item.id} : #{@item.try(:title)}"}
```

Comment:
I moved the view-item event tracking out of `show.html.haml` to separate the JavaScript code from the HTML code and to create consistency across all the Google Analytics event tracking code.